lib/active_fedora/file.rb

```diff
@@ -124,8 +131,13 @@ def content_changed?
       local_or_remote_content(false) != @ds_content
```

Comment:
@jcoyne ok, but that isn't evident in lines 107-114. To see `elsif` with nothing after it just looks wrong to me.